fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/config/FineractProperties.java

```diff
@@ -72,4 +75,21 @@ public static class FineractCorrelationProperties {
         private boolean enabled;
         private String headerName;
     }
+
+    @Getter
+    @Setter
+    public static class FineractPartitionedJobProperties {
```

Comment:
Not sure why this nested class is needed with only one attribute... why not just take that list and move one level up? Saves space and easier to read and follow.